migration/incremental/002-utils.sql

```diff
@@ -11,14 +11,24 @@ CREATE TABLE _prom_catalog.remote_commands(
 GRANT ALL ON SEQUENCE _prom_catalog.remote_commands_seq_seq TO current_user;
 
 CREATE OR REPLACE PROCEDURE _prom_catalog.execute_everywhere(command_key text, command TEXT, transactional BOOLEAN = true)
+    SET search_path = pg_catalog
 AS $func$
+DECLARE
+    _is_restore_in_progress boolean = false;
 BEGIN
     IF command_key IS NOT NULL THEN
        INSERT INTO _prom_catalog.remote_commands(key, command, transactional) VALUES(command_key, command, transactional)
        ON CONFLICT (key) DO UPDATE SET command = excluded.command, transactional = excluded.transactional;
     END IF;
 
     EXECUTE command;
+
+    -- do not call distributed_exec if we are in the middle of restoring from backup
```

Comment:
But what happens if we are? Do we expect people to dump/restore each node individually?

Reply:
As I understand it, we are going to work on multinode support as a fast follow up. Right @cevian ?

Reply:
Yes nodes are dump/restored individually